source/includes/administration/_notifications.md

```diff
@@ -0,0 +1,374 @@
+                        "id": "4840af1e-1d74-4b4f-bdda-6e04f4569142",
+                        "text": "system-category",
+                        "type": "title",
+                        "version": 1
+                    },
+                    {
+                        "language": "fr",
+                        "id": "0bc8f49c-8f65-4319-be57-ad70cdfe144d",
+                        "text": "en français",
+                        "type": "title",
+                        "version": 1
+                    }
+                ],
+                "icon": "fa fa-envelope",
+                "id": "2569f2cc-f1d7-4664-8439-173fedfab1c3",
+                "version": 1,
+                "updatedAt": "2020-10-29T19:28:22.000Z"
+            },
+            "splash": false,
+            "updatedAt": "2020-10-29T19:27:34.462Z",
+            "status": "published"
+        }
+    ]
+}
+```
+List the notifications configured for the organization.
+
+Optional Query Parameters | &nbsp;
+---------- | -----------
+`organization_id`<br/>*UUID* | The id of the organization we wish to display notifications for. When left blank, the current user's notifications will be shown.
+`language`<br/>*UUID* | Language of the notification. Values expected are of the form : "en" or "fr" as opposed to "english" or "french".
+
+Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The id of the notification.
+`authorId`<br/>*UUID* | The author's id.
+`authorFirstName`<br/>*string* | The author's first name.
+`authorLastName`<br/>*string* | The author's last name.
+`splash`<br/>*boolean* | When set to true, the notification will be displayed in the home screen until it reaches its deprecated date. Only one of splash and sticky can be set to true.
+`sticky`<br/>*boolean* | When set to true, the notification will be displayed upon logging in. Only one of splash and sticky can be set to true.
+`status`<br/>*string* | The status of the notification within it's lifecycle (draft, published, deprecated).
+`createdAt`<br/>*string* | The date on which the notification was created.
+`updatedAt`<br/>*string* | The date on which the notification was updated.
+`publishedAt`<br/>*string* | The date on which the notification was published.
+`depricatedAt`<br/>*string* | The date on which the notification will be deprecated.
+`version`<br/>*integer* | The notification's version.
+`category`<br/>*Array[[Notification Category](#administration-notification-categories)]* | The notification category that the notification belongs to.<br/>*includes*: `id`, `organizationId`, `createdAt`, `updatedAt`, `translations`, `icon`, `version`
+`translations`<br/>*Array[Object]* | The translations of the notification category's content.
+`translations.language`<br/>*string* | The language of the translation.
+`translations.text`<br/>*string* | The content of the notification.
+`translations.type`<br/>*string* | The content that we are translating (must be one of: title, body, description, url slug).
```

Comment:
This should only be title and body

Reply:
fixed 65d3185551293faa443a5e59617d81963fb934f1